src/components/tabs/tabs.yaml

```diff
@@ -8,7 +8,7 @@ examples:
         panel:
           html: |
             <h2 class="govuk-heading-l">Past day</h2>
-            <table class="govuk-table">
+            <table class="govuk-table govuk-!-margin-bottom-0">
```

Comment:
This works, just a question: In the conditional reveal I think we do this to any `:last-child`, should we consider doing that here too?